doc/man1/s_server.pod

```diff
@@ -8,116 +8,170 @@ s_server - SSL/TLS server program
 
 B<openssl> B<s_server>
 [B<-help>]
-[B<-port port>]
+[B<-port +int>]
```

Comment:
It is the convention (that you introduced) used in the apps to denote that an argument expects an integer greater than 0. If you do `openssl s_server -help` that is the help you get:

    $ ../util/shlib_wrap.sh ./openssl s_server -help
    Usage: s_server [options]
    Valid options are:
     -help                      Display this summary
     -port +int                 TCP/IP port to listen on for connections (default is 4433)

I'm just trying to remain consistent.